src/Integration/Binding/IRuleSetGenerationFileSystem.cs

```diff
@@ -10,16 +10,10 @@
 namespace SonarLint.VisualStudio.Integration.Binding
 {
     // Interface for testing purposes only
-    internal interface IRuleSetGenerationFileSystem
+    internal interface IRuleSetFileSystem
```

Comment:
It's just an "IRulesetSerializer" now; "FileSystem" suggests it does more than that, and suggests a relationship to IFileSystem and ISourceControlledFileSystem that doesn't exist.